lib/screens/b_quiz/tabs/quiz.dart

```diff
@@ -0,0 +1,171 @@
+import 'package:ecellapp/widgets/screen_background.dart';
+import 'package:flutter/material.dart';
+import 'package:circular_countdown_timer/circular_countdown_timer.dart';
+
+
+class Quiz extends StatefulWidget {
+  @override
+  State<Quiz> createState() => _QuizState();
+}
+
+class _QuizState extends State<Quiz> {
+  @override
+  Widget build(BuildContext context) {
+    double ratio = MediaQuery.of(context).size.aspectRatio;
+    double height = MediaQuery.of(context).size.height;
+    double width = MediaQuery.of(context).size.width;
+    double heightFactor = height / 1000;
+    return Scaffold(
+      body: Stack(children: [
+        ScreenBackground(elementId: 0),
+        Container(
+          width: width,
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.center,
+            mainAxisAlignment: MainAxisAlignment.start,
+            children: [
+              SizedBox(height: 30.0,),
+              Row(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  GradientText("BQUIZ",
+                      gradient: LinearGradient(
+                        colors: [
+                          C.bQuizGradient1,
+                          C.bQuizGradient2,
+                          C.bQuizGradient5,
+                        ],
+                        begin: Alignment.topLeft,
+                        end: Alignment.bottomRight,
+                      )),
+                  SizedBox(width: 20.0,),
+                  CircularCountDownTimer(
+                    duration: 30,
+                    initialDuration: 0,
+                    controller: CountDownController(),
+                    width: MediaQuery.of(context).size.width / 7,
+                    height: MediaQuery.of(context).size.height / 10,
+                    ringColor: Colors.grey[300],
+                    fillColor: Colors.purpleAccent[100],
+                    backgroundColor: Colors.purple[500],
+                    strokeWidth: 10.0,
+                    strokeCap: StrokeCap.round,
+                    textStyle: TextStyle(
+                        fontSize: 25.0, color: Colors.white, fontWeight: FontWeight.bold),
+                    textFormat: CountdownTextFormat.S,
+                    isReverse: true,
+                    isReverseAnimation: false,
+                    isTimerTextShown: true,
+                    autoStart: true,
+                    onStart: () {
+                      debugPrint('Countdown Started');
+                    },
+                    onComplete: () {
+                      debugPrint('Countdown Ended');
+                    },
+                  ),
+                ],
+              ),
+              Container(
+                height: height/3.5,
+                child: Text("Question",style: TextStyle(
+                  fontSize: 25.0,
+                  color: Colors.white,
+                ),),
+              ),
```

Comment:
You can here probably have a flag variable in Question model which indicates 1-> text 0->image based quiz
Then it will show image else would switch to text. 
Use dart property here  flag?TextQuizWidget():ImageQuizWidget

You can follow [this](https://github.com/ECellNitrr/ecell-main-server/blob/5ffd978769946b2efd8289bc9a9a86228ed6ace6/bquiz/models.py#L41) model for refrence:

![image](https://user-images.githubusercontent.com/16835617/194722935-adadb7d0-e14f-418b-9cae-545db38b5318.png)